### PR TITLE
modify some files for release 2.3.0 version

### PR DESCRIPTION
**Pull request by wangfakang**

### CODE REVIEW

File: CHANGES.te [hunk 1]
```diff
@@ -1,3 +1,58 @@
+Changes with Tengine 2.3.0                                         20 MAR 2019
+
+    *) Feature: Add proxy_connect module support for the CONNECT
```
Comment (chobits) [inline CHANGES.te]: Add -> Added

File: CHANGES.te [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+       in dyups and session_sticky modules. (wangfakang)
+    *) Bugfix: Fixed compilation error of limit_req、http2 module. (hongxiaolong)
+    *) Bugfix: Fixed removes the Unix domain socket file
+       when pipe proc close listen socket. (wangfakang)
+    *) Bugfix: Fixed compatibility for --with-openssl
+       and --with-openssl-async. (mrpre)
+    *) Bugfix: Fixed regression bug that ngx_http_top_intput_body_filter
+       was removed. (chobits)
+    *) Bugfix: Fixed reuse_port and accept_mutex conflict. (innomentats)
+    *) Bugfix: Fixed tengine build fails when using gcc7 compiler. (wangfakang)
```
Comment (chobits) [inline CHANGES.te]: fails -> failure

File: CHANGES.cn [hunk 3]
```diff
@@ -1,3 +1,37 @@
+Tengine 2.3.0 [2019-03-20]
+
+* Feature: 新增proxy_connect模块，可以支持HTTP的CONNECT方法 [chobits]
+* Feature: Stream模块server配置块支持server_name指令 [mrpre]
+* Feature: reqstat模块新增req_status_lazy指令 [taoyuanyuan]
+* Feature: 新增http2指令，可以在server配置块中控制http2开关 [jinjiu]
+* Feature：新增ssl_handshake_time变量，统计SSL握手时间 [jinjiu]
+* Feature: limit_req_zone指令的rate参数支持变量模式 [Alaaask]
```
Comment (chobits) [inline CHANGES.cn]: `:` -> `：`

**Comment by chobits** [comment]
link to https://github.com/alibaba/tengine/issues/1196

File: CHANGES.te (hunk 2 above)
Comment (wangfakang) [reply CHANGES.te]: fixed.

File: CHANGES.te (hunk 1 above)
Comment (wangfakang) [reply CHANGES.te]: fixed

File: CHANGES.cn (hunk 3 above)
Comment (wangfakang) [reply CHANGES.cn]: fixed

File: CHANGES.cn [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+* Change: 对core代码的修改全部使用宏区分开 [chobits、wangfakang、fankeke、hongxiaolong、imkeeper]
+* Change: 恢复accpte_filter功能 [wangfakang]
+* Bugfix: 修复dyups模块在和高版本OpenSSL一起编译的报错问题 [wangfakang]
+* Bugfix: 修复dyups模块init_number初始化逻辑 [FengXingYuXin]
+* Bugfix: 修复rollback日志进程在reload时可能导致日志写到滚动后的文件里面 [MengqiWu]
+* Bugfix: 修复ssl_verify_client_exception指令导致的coredump问题 [chobits]
+* Bugfix: 修复dyups、session_sticky模块在升级core代码后导致的coredump问题 [wangfakang]
+* Bugfix: 修复limit_req、http2模块的编译错误 [hongxiaolong]
+* Bugfix: 修复日志pipe进程在关闭监听socket时误删除监听Unix domain文件 [wangfakang]
+* Bugfix: 修复静态加载OpenSSL库时导async openssl无法检测 [mrpre]
```
Comment (chobits) [inline CHANGES.cn]: > * Bugfix: 修复静态加载OpenSSL库时导async openssl无法检测 [mrpre]

"导" 这个字没理解

Comment (wangfakang) [reply CHANGES.cn]: fixed.

**Pull request merged**